docs/docs/recipes/migrations/README.mdx

```diff
@@ -0,0 +1,89 @@
+---
+sidebar_position: 14
+---
+
+import Availability from '@components/Availability';
+
+# 🔄 Migrate to Logto
+
+<Availability cloud oss />
```

Comment:
nit: Should we only mark it as `cloud`?

Reply:
The create user API is also available in OSS, so this is not cloud only feature.